udp/client/clientconn.go

```diff
@@ -195,8 +201,10 @@ func (cc *ClientConn) Do(req *pool.Message) (*pool.Message, error) {
 		req.UpsertMessageID(cc.getMID())
 		return cc.do(req)
 	}
+	msgType := req.Type()
 	bwresp, err := cc.blockWise.Do(req, cc.blockwiseSZX, cc.session.MaxMessageSize(), func(bwreq blockwise.Message) (blockwise.Message, error) {
 		req := bwreq.(*pool.Message)
+		req.SetType(msgType)
```

Comment:
At blockwise it is recommended to use `.Confirmable` messages:
```
   specification minimizes the constraints it adds to those base
   exchanges; however, not all variants of using CoAP are very useful
   inside a block-wise transfer (e.g., using Non-confirmable requests
   within block-wise transfers outside the use case of Section 2.8 would
   escalate the overall non-delivery probability).
   ```
We need to change update `get,post,delete,put` operations to use `Confirmable` type by default.